docs/schema/reference.md

```diff
@@ -403,6 +405,12 @@ Notes:
 
 * The `dateModified` field should be changed whenever the progress towards a milestone is reviewed, and the `status` either updated, or re-confirmed. 
 
+```{admonition} Planned payments
+:class: seealso
+
+To learn about how to represent planned payments, see the [Delivery and payment data worked example](../guidance/map/milestones.md#delivery-and-payment-data)
+```
+
```

Comment:
Move this and the notes above the table. Remove the existing "Notes:" and change the bullet to a simple paragraph. (This follows from an another issue, where we want all content except the extensions list above the jsonschema directive, as otherwise it's easy to miss.)

I think we might start using a simpler format of "seealso ... How to [do X]" when linking to guidance.

```suggestion
```{seealso}
[How to represent planned payments](../guidance/map/milestones.md#delivery-and-payment-data)
```
```

Reply:
Changes made in https://github.com/open-contracting/standard/pull/1434/commits/d639e15b1495ffe05d50c0ef71725dfa572b6428

Both suggestions sound good. Shall I open an issue to go through `reference.md` and make those changes?

Reply:
I think they can be combined with https://github.com/open-contracting/standard/issues/1297